<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>VRCEasyTrans</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="launchButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>520</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>       启动       </string>
    </property>
   </widget>
   <widget class="QFrame" name="oscFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VRChat连接</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>端口号：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>OSC 主机：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="oscPortInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9000</string>
     </property>
     <property name="maxLength">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="oscHostInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
     <property name="maxLength">
      <number>60</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="deepseekFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Deepseek</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ApiKeyInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>API密钥：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QComboBox" name="deviceCombo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>音频输入设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音频输入设备：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>断句时长/ms：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>静音阈值：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="silentTimeInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">1000</string>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="cursorPosition">
      <number>4</number>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="silentTimeInput_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">1.5</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="cursorPosition">
      <number>3</number>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="languageCombo">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>英语(EN)</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>英语(EN)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>日语(JP)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>韩语(KO)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>俄语(RU)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>法语(FR)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西班牙语(ES)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>目标语言：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="deepseekFrame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>百度云</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="baiduAPIKeyInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>API Key：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="baiduSecretKeyInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Secret Key：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>deepseekFrame</zorder>
   <zorder>oscFrame</zorder>
   <zorder>launchButton</zorder>
   <zorder>frame_2</zorder>
   <zorder>languageCombo</zorder>
   <zorder>label_6</zorder>
   <zorder>deepseekFrame_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
